README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# Weather App :sunny: :zap: :rainbow: :cloud_with_rain: :tornado:
```

Comment:
A screenshot would be nice here! Also what the test script is so its obvious that you have written tests in the first instance